dts/arm/st/stm32f407.dtsi

```diff
@@ -4,10 +4,15 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <st/stm32f407-pinctrl.dtsi>
 #include <st/stm32f411.dtsi>
 
 / {
 	soc {
+		pinctrl: pin-controller {
+			reg = <0x40020000 0x2800>;
+		};
```

Comment:
Should we have:

			compatible = "st,stm32-pinmux";
 +			#address-cells = <1>;
 +			#size-cells = <1>;
 +			reg = <0x40020000 0x1C00>;
 +